test/e2e/extensions_crud_test.go

```diff
@@ -0,0 +1,166 @@
+package e2e
+
+import (
+	"testing"
+
+	"github.com/openshift/console-operator/pkg/api"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	consolev1 "github.com/openshift/api/console/v1"
+	operatorsv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/console-operator/test/e2e/framework"
+)
+
+func setupExtensionsTest(t *testing.T) (*framework.ClientSet, *operatorsv1.Console) {
+	return framework.StandardSetup(t)
+}
+
+func cleanupExtensionsTest(t *testing.T, client *framework.ClientSet) {
+	framework.StandardCleanup(t, client)
+}
+
+func TestCreateCLIDownloadLink(t *testing.T) {
+	client, _ := setupExtensionsTest(t)
+	defer cleanupExtensionsTest(t, client)
+
+	download := &consolev1.ConsoleCLIDownload{
+		ObjectMeta: v1.ObjectMeta{
+			Name:      "test-cli-download",
+			Namespace: api.OpenShiftConsoleNamespace,
+		},
+		Spec: consolev1.ConsoleCLIDownloadSpec{
+			DisplayName: "test",
+			Description: "test",
+			Links: []consolev1.CLIDownloadLink{{
+				Text: "download test",
+				Href: "https://example.com",
+			}},
+		},
+	}
+
+	download, err := client.ConsoleCliDownloads.Create(download)
+	if err != nil {
+		t.Fatalf("%v", err)
```

Comment:
nit: let's make this one more noisy:
```suggestion
		t.Fatalf("could not create ConsoleCliDownloads custom resource: %v", err)
```